### PR TITLE
Use the 'correct' windows platform version

### PR DESCRIPTION
**Pull request by BrennanConroy**
Fixes https://github.com/aspnet/AspNetCore-Internal/issues/1780

### CODE REVIEW

**Review by jkotalik — APPROVED** [review]

File: src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <OutDir>$(OutDir)lib\</OutDir>
     <IntDir>$(Configuration)\lib\</IntDir>
     <ConfigurationType>StaticLibrary</ConfigurationType>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```
Comment (Eilon) [inline src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj]: Yet another "off by 0.0.629.0" error... so typical!

Comment (BrennanConroy) [reply src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj]: There are two problems in programming...

**Pull request merged**